tests/envs/HEROKU_APP_NAME

```diff
@@ -0,0 +1 @@
+website-develop
```

Comment:
Yep, they'll change depending on what Heroku sends, this one's just for testing though.